ffcx/options.py

```diff
@@ -20,6 +20,7 @@
 logger = logging.getLogger("ffcx")
 
 FFCX_DEFAULT_OPTIONS = {
+    "cuda": (bool, False, "generate CUDA wrapped versions of tabulate tensor functions", None),
```

Comment:
Better strategy might be to always generate it and let the C preprocessor include it if CUDA is available.

Reply:
This should clarify that they are NVRTC wrappers, rather than CUDA wrappers (which I associate with `nvcc`).